howto/diagnose-with-tracing/open-telemetry-collector.md

```diff
@@ -0,0 +1,254 @@
+        component: otel-collector
+    spec:
+      containers:
+      - name: otel-collector
+        image: otel/opentelemetry-collector-contrib-dev:latest
+        command:
+          - "/otelcontribcol"
+          - "--config=/conf/otel-collector-config.yaml"
+        resources:
+          limits:
+            cpu: 1
+            memory: 2Gi
+          requests:
+            cpu: 200m
+            memory: 400Mi
+        ports:
+          - containerPort: 55678 # Default endpoint for Opencensus receiver.
+        volumeMounts:
+          - name: otel-collector-config-vol
+            mountPath: /conf
+          #- name: otel-collector-secrets
+          #  mountPath: /secrets
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 13133
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 13133
+      volumes:
+        - configMap:
+            name: otel-collector-conf
+            items:
+              - key: otel-collector-config
+                path: otel-collector-config.yaml
+          name: otel-collector-config-vol
+EOF
+
+```
+
+
+Set up a Dapr exporter component
+```
+kubectl apply -f- <<EOF
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: appconfig
+  namespace: default
+spec:
+  # Configure Dapr tracing.
+  tracing:
+    samplingRate: "1"
+---
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: native
+  namespace: default
+spec:
+  type: exporters.native
+  metadata:
+  - name: enabled
+    value: "true"
+  - name: agentEndpoint
+    value: "otel-collector.default.svc.cluster.local:55678"
+EOF
+```
+
+### Deploy your app with tracing
+
+When running in Kubernetes mode, apply the configuration by adding a
+`dapr.io/config` annotation to the container that you want to
+participate in the distributed tracing, as shown in the following
+example:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  ...
+spec:
+  ...
+  template:
+    metadata:
+      ...
+      annotations:
+        dapr.io/enabled: "true"
+        dapr.io/app-id: "calculator-front-end"
+        dapr.io/app-port: "8080"
+        dapr.io/config: "tracing"
+```
+
+Some of our quickstarts like [distributed
+calculator](https://github.com/dapr/quickstarts/tree/master/distributed-calculator)
+already configure these settings, so if you are trying those no
+additional settings are needed.
+
+That's it! There's no need include any SDKs or instrument your
+application code. Dapr automatically handles the distributed tracing
+for you.
+
+> **NOTE**: You can register multiple exporters at the same time, and
+> the tracing logs are forwarded to all registered exporters.
+
+Deploy and run some applications. After a few minutes, you should see
+tracing logs appearing in your Application Insights resource. You can
+also use **Application Map** to examine the topology of your services,
+as shown below:
+
+![Application map](../../images/azure-monitor.png)
```

Comment:
Can you see the same image when you use OTEL ? I would update this image as well..

Reply:
Done